packages/shared/src/lib/placeholders.ts

```diff
@@ -1,20 +1,48 @@
-import { getCountryName, getLocalCurrencySymbol } from './geolocation';
+import { getCountryName, getLocalCurrencySymbol, countryCodeToCountryGroupId } from './geolocation';
+import { Prices } from '../types/prices';
 
 // we have to treat %%ARTICLE_COUNT%% placeholders specially as they are replaced
 // with react components, not a simple text substitution
 export const replaceNonArticleCountPlaceholders = (
     content: string | undefined,
     countryCode?: string,
+    prices?: Prices,
 ): string => {
     if (!content) {
         return '';
     }
 
-    content = content.replace(/%%CURRENCY_SYMBOL%%/g, getLocalCurrencySymbol(countryCode));
+    const localCurrencySymbol = getLocalCurrencySymbol(countryCode);
+
+    content = content.replace(/%%CURRENCY_SYMBOL%%/g, localCurrencySymbol);
 
     const countryName = getCountryName(countryCode) ?? '';
     content = countryName ? content.replace(/%%COUNTRY_NAME%%/g, countryName) : content;
 
+    // Pricing templates
+    const countryGroupId = countryCodeToCountryGroupId(countryCode);
+    const localPrices = prices && prices[countryGroupId] ? prices[countryGroupId] : null;
+
+    if (localPrices != null) {
+        const { GuardianWeekly, Digisub } = localPrices;
+
```

Comment:
this is all fine, but I'm going to add a card to make this function more efficient as it's now scanning the same string many times